contentcuration/contentcuration/forms.py

```diff
@@ -251,6 +251,8 @@ def save(self, user):
 class PreferencesSettingsForm(forms.Form):
     # TODO: Add language, audio thumbnail, document thumbnail, exercise thumbnail, html5 thumbnail once implemented
     author = forms.CharField(required=False, label=_('Author'), widget=forms.TextInput(attrs={'class': 'form-control setting_input'}))
+    aggregator = forms.CharField(required=False, label=_('Aggregator'), widget=forms.TextInput(attrs={'class': 'form-control setting_input'}))
```

Comment:
Added default aggregator and provider to preference form

Reply:
Aggregator = who put the content together and provider = who distributed the content (see [models.py](https://github.com/jayoshih/content-curation/blob/64359945d6806ee323f07a9ff7831dc5d7a7f519/contentcuration/contentcuration/models.py#L636))